data/fixtures/scopes/textual/boundedParagraph.scope

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
These scope tests didn't really turn out to well since the position for our recorder is 0,0. I wonder if we should do something like `$0` in the future to indicate position?
For now I think it's fine to just have the files there and we can look into an improvement to our scope recorder in the future.

Reply:
This is a bug in your implementation. You're not yielding nested scopes

Reply:
I punted on that. It made multiple test fail in unexpected ways. I don't know how it even should behave. 
Our test recorder should still be able to catch the scope that is there was my point.